vue/components/configurator/configurator.vue

```diff
@@ -221,6 +221,14 @@ export const Configurator = {
             this.$store.commit("current_frame", frame);
         });
 
+        this.configurator.bind("highlighted_part", part => {
+            this.$bus.trigger("highlighted_part", part);
+        });
+
+        this.configurator.bind("lowlight", () => {

```

Comment:
If this event's counterpart is `highlighted_part` than this should be `lowlighted_part`.

Reply:
That´s what I was doing but then I notice that the the lowlight event "lowlights" the whole configurator, meaning that if I went from hovering the sole to the front, even though the sole lowlights and the front highlights, the _lowlight_ event will not be triggered.

Reply:
Ok that makes sense. `lowlighted` then?